tests/camera/CMakeLists.txt

```diff
@@ -8,8 +8,11 @@ target_link_libraries(CameraTest
 add_test(NAME CameraTest COMMAND CameraTest)
 
 set(_env_vars)
-list(APPEND _env_vars "GZ_SIM_SYSTEM_PLUGIN_PATH=$<TARGET_FILE_DIR:gz-sim-yarp-camera-system>")
-list(APPEND _env_vars "LIBGL_ALWAYS_SOFTWARE=1")
+list(APPEND _env_vars
+  "GZ_SIM_SYSTEM_PLUGIN_PATH=$<TARGET_FILE_DIR:gz-sim-yarp-camera-system>"
+  "GZ_SIM_RESOURCE_PATH=${GZ_SIM_RESOURCE_PATH}:${CMAKE_CURRENT_SOURCE_DIR}"
+  "LIBGL_ALWAYS_SOFTWARE=1"
```

Comment:
Not directly related to this PR, but I just noticed: this was probably added since the original authors of these code were working on WSL, and this was required to get gz-sim to work on WSL, but this is not necessary anymore: https://community.gazebosim.org/t/wslg-with-gpu-support-available-on-latest-version-of-gazebo-garden-and-harmonic/2360 . Anyhow, we can remove it in a subsequent PR.

Reply:
Ok I will clean the code from this variable in one of the next PRs, as described in #40